src/main/resources/templates/fragments/navbar.html

```diff
@@ -42,11 +42,11 @@
                     </a>
                 </li>
 
-                <li sec:authorize="isAuthenticated()">
-                    <a th:href="@{/cart}">
-                        <span class="glyphicon glyphicon-shopping-cart"></span>  Cart
-                    </a>
-                </li>
+<!--                <li sec:authorize="isAuthenticated()">-->
```

Comment:
nu vrem sa aratam cartul unui utilizator nelogat, puteai doar sa modifici din cart in shopping cart